lib/src/main/java/org/team1540/rooster/drive/pipeline/CTREOutput.java

```diff
@@ -0,0 +1,118 @@
+package org.team1540.rooster.drive.pipeline;
```

Comment:
Not sure if I can really review this adequately, there seem to be fairly reasonable gaps in my understanding of the CTRE controllers that could be problematic.